tv_apps/app-deck/js/app_deck.js

```diff
@@ -139,12 +139,19 @@
           this._spatialNavigator.focus(elem);
         }
         elem.focus();
+        this._scrollTo(elem);
```

Comment:
I'll like to have some comment here. Because code reader might get confused when they read through here. What you said on https://bugzilla.mozilla.org/show_bug.cgi?id=1120381#c1 is suitable here I think.